archunit/src/test/java/com/tngtech/archunit/library/GeneralCodingRulesTest.java

```diff
@@ -1,10 +1,15 @@
 package com.tngtech.archunit.library;
 
 import com.tngtech.archunit.core.importer.ClassFileImporter;
-import com.tngtech.archunit.library.testclasses.packages.correct.ImplementationClassWithCorrectPackage;
-import com.tngtech.archunit.library.testclasses.packages.incorrect.ImplementationClassWithWrongTestClassPackage;
-import com.tngtech.archunit.library.testclasses.packages.incorrect.wrongsubdir.ImplementationClassWithWrongTestClassPackageTest;
-import com.tngtech.archunit.library.testclasses.packages.incorrect.wrongsubdir.ImplementationClassWithWrongTestClassPackageTestingScenario;
+import com.tngtech.archunit.library.testclasses.packages.correct.customsuffix.ImplementationClassWithCorrectPackageCustomSuffix;
+import com.tngtech.archunit.library.testclasses.packages.correct.defaultsuffix.ImplementationClassWithCorrectPackage;
+import com.tngtech.archunit.library.testclasses.packages.correct.onedirmatching.ImplementationClassWithOneTestPackageMatchingOutOfTwo;
+import com.tngtech.archunit.library.testclasses.packages.correct.notest.ImplementationClassWithoutTestClass;
+import com.tngtech.archunit.library.testclasses.packages.incorrect.nonedirmatching.ImplementationClassWithMultipleTestsNotMatchingImplementationClassPackage;
```

Comment:
Shouldn't it be `nodirmatching`? `nonedirmatching` sounds strange to me :thinking:

Reply:
Fixed 🚀